Cargo.toml

```diff
@@ -16,7 +16,7 @@ readme = "README.md"
 
 [dependencies]
 enumflags2 = "0.7"
-libc = "0.2"
+libc = "0.2.134"
```

Comment:
The earliest version is actually `0.2.133`, I've changed it to make support a tiny bit better even.

> It's a pity that the libc crate is using such versioning because we cannot have any guarantee that the used crate will have these flags

I'm not sure what you mean about this. The negotiation for versions should only ever go up, so it shouldn't be possible to pull landlock and end up with a version under `0.2.133` (after my latest change). So I think we can guarantee that the used crate will have these flags?

Reply:
[Correct](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio), I was thinking about the `0.*` exception but it is unrelated.